deployment/hasura/metadata/databases/tables/ui/derivation_group_comp.yaml

```diff
@@ -0,0 +1,30 @@
+table:
+  name: derivation_group_comp
+  schema: ui
+configuration:
+  custom_name: "derivation_group_comp"
+object_relationships: # nothing for the arrays, can't grab string values and associate as ids in hasura. but shouldn't need to...
```

Comment:
Please clarify which array relationship you wanted to set up? Also remove this comment--it does not provide enough information to be useful to someone reading this code.

Reply:
The comment has been removed as it was a source of confusion. This was not in reference to an array relationship, but rather an array type (a list of `{source_key, derivation_group_name, number_of_contained_events}`) in SQL. The comment was simply denoting that there was no easy way to create a Hasura relationship off of that.